velox/exec/Operator.cpp

```diff
@@ -299,21 +312,30 @@ RowVectorPtr Operator::fillOutput(vector_size_t size, BufferPtr mapping) {
       std::move(columns));
 }
 
+OperatorStats Operator::stats(bool clear) {
+  auto lockedStats = stats_.wlock();
```

Comment:
```
  OperatorStats ret;
  if (clear) {
    auto lockedStats = stats_.wlock();
    ret = *lockedStats;
    lockedStats->clear();
  } else {
    ret = *stats_.rlock();
  }
   return ret;
```

Reply:
@xiaoxmeng OperatorStats does not have default constructor. It requires few members at construction time.